Packs/CofenseTriage/.pack-ignore

```diff
@@ -12,4 +12,53 @@ ignore=IM111
 
 [known_words]
 cofense
-
+cofens
```

Comment:
Were these just getting picked up by the secrets detection?

Reply:
It was detected by the doc-review command.